lib/Twig/Extension/Profiler.php

```diff
@@ -11,11 +11,11 @@
 
 class Twig_Extension_Profiler extends Twig_Extension
 {
-    private $actives;
+    private $actives = [];
```

Comment:
the other way would add an extra step as well: first it's initialized as null and then changed to an array.